### PR TITLE
Feature/main page styling ii

### PR DESCRIPTION
**Pull request by Matt-Roden**
# Description drawing

- updated the styling of the pokedex grid display
- added hovers, new color themes, font styles, and box shadows
- Still need to test in Cypress

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [x] local host 3000
- [ ] console.log()
- [x] dev tools
- [ ] cypress testing

# Checklist:

- [x] My code follows the style guidelines of this project/linted
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/PokemonCard/PokemonCard.js [hunk 1]
```diff
@@ -6,11 +6,11 @@ const PokemonCard = (props) => {
     return (
         <section className="pokemon-card">
             <h2>{ props.name }</h2>
-            <img src={ props.image } className="pokemon-i" alt={`Pokemon ${ props.name }`} />
+            <img src={ props.image } className="pokemon-image" alt={`Pokemon ${ props.name }`} />
```
Comment (joanafbrito) [inline src/components/PokemonCard/PokemonCard.js:8]: I did image before but I got an alert message in the console saying that it is redundant to use image in a image tag, the readers would read twice, that's the reason why it is -i

File: src/components/App/App.css [hunk 2]
```diff
@@ -4,3 +4,8 @@
   flex-direction: column;
 
 }
+/* .call-to-action-text {
```
Comment (hayleyw7) [inline src/components/App/App.css:7]: Amazing font.

**Review by joanafbrito — APPROVED** [review]
Good job! 
Thank you , my only observation would be about the image on PokemonCard line 9

File: src/components/PokedexGrid/PokedexGrid.css [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
-    background: url("https://lh3.googleusercontent.com/ohXfK_gejGdGoxN7Vzf2hE9EWsvXhR9AnPlRqylsbhSFn1hU2pDZ4dkG8Q8Wj8cd4g9hVGShd2X7qBSTkbWiX-2UmhH44u_-M4xpO2nMBYsi0ymKeMpfZFOMWXtTJW28YdTjVquwquv9pTTCBFpTnG_C8euEUgO08mXO63oDcqIN9bmSvh5B8MUDw2MJ_oYAMJoCi8UeI9nvfs7z3UbcQQPNYHyXgi0wKW9aMmz8ogaRUPHf5Z7tKv9Am8my9XsleXNzeZjlTY7qobFffw5xwpNBUGNYdPWfkF-XWQyhrcJsESwGFHUlFDtVvU_V6IOgHA-jZmnmu8e3_wBA50VTO713U-DoDMjhZliK1qSUoR9FBxcPE7Z_O3kVk9v5qFUgwO2wmybVKDmH9gn55vDnBEiHCr-0cfKJgJHYlgIFm3fwguGYtIEpYK5ucGAxjz_NPcM5fw9BNQ-M67dyM5eY9Tdav198oMNTAJLKGmdaEG4v8UqRsxk5hyFhOWeufn6vuaG2TrxB9_kDR5gaYroneuDaYB9CySvKyXo_AmqA3po7mGl8W0HBphFHzOPkfFqyGdpTfeJNurtKuuV99cb57kHxwieMwWCPwUNRFzZwOByzNyfcPkeVMgy8z7tpYj6Eb9SUpt8Y-6j9-Qj9lfxnHJNTZ0CTzxvJ-N_Mwya-YA=w1135-h638-no") no-repeat;
-    background-size: cover;
-    height: 65vh;
-    margin-left: 5%;
-    margin-right: 5%;
-    /* margin: 0; */
-
+    font-family: 'PocketMonk', sans-serif;
+    background: white;
+    height: 455px;
```
Comment (hayleyw7) [inline src/components/PokedexGrid/PokedexGrid.css:10]: All of this looks so great.

File: src/components/PokemonCard/PokemonCard.css [hunk 4]
```diff
@@ -1,10 +1,25 @@
 .pokemon-card {
```
Comment (hayleyw7) [inline src/components/PokemonCard/PokemonCard.css:1]: Fun!

**Review by hayleyw7 — APPROVED** [review]

**Pull request merged**